src/sqarql-query.js

```diff
@@ -15,7 +17,18 @@ async function sparqlQuery(query) {
 			return json;
 		}
 	} catch(error) {
-		throw ['Fetch Error :-S', error];
+		if (attempt > 5) {
+			throw ['Fetch Error :-S', error];
+		}
+		
+		const delay = Math.floor(Math.random() * 1000 * attempt)
+		console.warn(`Sparql Query failed, retring in ${delay / 1000} seconds`)
```

Comment:
technically this log is not correct anymore (does not take into account the attempt).

Reply:
what do you mean? should the log state which attempt it was? Or what do you propose?

Reply:
I mean that now that the log will now say `retrying in x seconds` but would actually retry in `x*attempt` seconds

Reply:
but, `attempt` is already part of the calculation one line above

Reply:
ah, sorry - missed that!